### PR TITLE
Networking stuff

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Not packet stuff
Makes names more meaningful and more aligned with yarn :yeef:

### CODE REVIEW

**Comment by BoogieMonster1O1** [comment]
also i dropped invalid mappings so ignore any rearrangements

File: mappings/net/minecraft/network/ClientConnection.mapping [hunk 1]
```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/class_117 net/minecraft/network/ClientConnection
+	FIELD field_1274 LOCK Ljava/lang/Object;
+	FIELD field_1275 READ_THREADS I
+	FIELD field_1276 WRITE_THREADS I
+	FIELD field_1280 lock Ljava/lang/Object;
+	FIELD field_1281 socket Ljava/net/Socket;
+	FIELD field_1282 socketAddress Ljava/net/SocketAddress;
+	FIELD field_1283 dataInput Ljava/io/DataInputStream;
+	FIELD field_1284 dataOutput Ljava/io/DataOutputStream;
```
Comment (valoeghese) [inline mappings/net/minecraft/network/ClientConnection.mapping:9]: input and output would be more concise

File: mappings/net/minecraft/network/ClientConnection.mapping [hunk 2]
```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/class_117 net/minecraft/network/ClientConnection
```
Comment (valoeghese) [inline mappings/net/minecraft/network/ClientConnection.mapping:1]: ConnectionHandler

File: mappings/net/minecraft/server/network/ServerLoginPacketHandler.mapping [hunk 3]
```diff
@@ -1,11 +1,16 @@
-CLASS net/minecraft/class_10 net/minecraft/server/network/PendingConnection
+CLASS net/minecraft/class_10 net/minecraft/server/network/ServerLoginPacketHandler
```
Comment (valoeghese) [inline mappings/net/minecraft/server/network/ServerLoginPacketHandler.mapping:1]: PendingConnection

File: mappings/net/minecraft/server/network/ServerPlayPacketHandler.mapping [hunk 4]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_11 net/minecraft/server/network/ServerPlayPacketHandler
```
Comment (valoeghese) [inline mappings/net/minecraft/server/network/ServerPlayPacketHandler.mapping:1]: ServerPlayerConnection

**Review by valoeghese — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/network/ClientConnection.mapping (hunk 2 above)
Comment (valoeghese) [reply mappings/net/minecraft/network/ClientConnection.mapping:1]: PacketDispatcher

**Comment by valoeghese** [comment]
ok nvm wait until tomorrow or later this week

**Review by valoeghese — APPROVED** [review]

File: mappings/net/minecraft/network/ClientConnection.mapping (hunk 2 above)
Comment (valoeghese) [reply mappings/net/minecraft/network/ClientConnection.mapping]: nvm Connection

**Comment by valoeghese** [comment]
let me know whan u test it

**Pull request merged**